tiledb/api/src/error.rs

```diff
@@ -26,107 +29,6 @@ impl Drop for RawError {
     }
 }
 
-#[derive(Clone, Debug)]
-pub enum DatatypeErrorKind {
```

Comment:
Some variants are moved to `tiledb_common::datatype::Error` and others are now just `Error` variants.